org-formation/070-guard-duty/guardduty-s3-malware-protection.yaml

```diff
@@ -0,0 +1,110 @@
+# Setup AWS Guardduty Malware Protection for S3 to scan and apply a predefined "GuardDutyMalwareScanStatus" tag all
+# S3 objects with possible tag values ["NO_THREATS_FOUND","THREATS_FOUND","UNSUPPORTED","FAILED"]
+# https://docs.aws.amazon.com/guardduty/latest/ug/tag-based-access-s3-malware-protection.html
+#
+# Note: The bucket being scanned should contain a bucket policy to restrict access to Malicious objects.
+# https://docs.aws.amazon.com/guardduty/latest/ug/tag-based-access-s3-malware-protection.html#apply-tbac-s3-malware-protection
+
+AWSTemplateFormatVersion: 2010-09-09
+Description: Guardduty Malware protection plan for S3 bucket
+Parameters:
+  BucketName:
```

Comment:
I'm not sure if I'm reading this right, is this the bucket being scanned and so we would need to deploy a separate stack for every bucket we want to scan?

Reply:
I agree, additionally we have different buckets in different accounts. I would first deploy and test this to the synapse dev account.

Reply:
@marcomarasca instead of deploying with org-formation you can take the `guardduty-s3-malware-protection.yaml` template and deploy it to synapse dev with the synapse java deployer.

Reply:
I wouldn't want to have a different deployment for dev, we have to test if the integration works.

Reply:
I think this template is ready for you to test now @marcomarasca.  Just pass in the bucket and topic name and it should be good to go.